src/components/error/_macro.spec.js

```diff
@@ -36,11 +36,23 @@ describe('macro: error', () => {
       ),
     );
 
-    expect(
-      $('.ons-panel__error')
-        .text()
-        .trim(),
-    ).toBe('Example error text.');
+    expect($('.ons-panel__error').text().trim()).toBe('Example error text.');
+  });
+
+  it('has the provided `text` if an array was provided', () => {
```

Comment:
I think this test should have a more descriptive description. Something like "when the 'text' param is set to an array type, then all error messages in the array are shown".
I think we may need a card to look at the way the tests are structured and use Given-When-Then to write the descriptions or something similar